### ISSUE TITLE
py_exprs test fails in parallel on Windows

### ISSUE THREAD

**Issue by aowen87** [aowen87/FOO#10]
I run the windows equivalent of './runtest -n 1 --lessverbose -m parallel tests/hybrid/py_exprs.py'

Here is the message printed out to the command window:

VisIt: Error - Pseudocolor:  (ExpressionException)
viewer.exe: The 'py' expression failed because avtPythonExpression::ProcessArguments Error - Failed to initialize the python filter environment.
Python Environment Error:
<type 'exceptions.AttributeError'>'module' object has no attribute 'init'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AttributeError: 'module' object has no attribute 'init'

I have added the test to the skip list until it can be fixed.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Kathleen Biagas
Original creation: 08/04/2017 12:07 pm
Original update: 03/07/2018 03:21 pm
Ticket number: 2881
